main/EditText/index.tsx

```diff
@@ -468,13 +447,11 @@ function EditText(props: Props): ReactElement {
               multiline={multiline}
             />
           </StyledRowContent>
-          {
-            errorText
-              ? <StyledInvalidText testID={errorTestID} style={errorTextStyle}>
-                {errorText}
-              </StyledInvalidText>
-              : null
-          }
+          {errorText ? (
```

Comment:
It can be `{errorText && ...}` as you already did.

Reply:
Thank you for your reviewing. When I try to change that code to && syntax, It make an error.   I can't understand why that happened. Could you check this error? Thank you in advance. :)


<img width="409" alt="Screen Shot 2020-06-17 at 10 03 38" src="https://user-images.githubusercontent.com/58724686/84843073-e5bf2300-b081-11ea-91bc-44504cb8a281.png">

<img width="743" alt="Screen Shot 2020-06-17 at 10 02 53" src="https://user-images.githubusercontent.com/58724686/84843060-dcce5180-b081-11ea-8732-c0b7903248ae.png">